packages/cli/generation/ir-generator/src/converters/type-declarations/convertExampleType.ts

```diff
@@ -362,7 +375,7 @@ function convertPrimitiveExample({
     return PrimitiveTypeV1._visit(typeBeingExemplified, {
         string: () => {
             if (typeof example !== "string") {
-                throw new Error("Example is not a string");
+                throw new Error(`Error converting to string primitive: example ${example} is not a string`);
```

Comment:
nit: can we make these error messages a little more concise? 

`Expected string. Received ${example}`